src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -74,6 +75,8 @@
   --pf-c-about-modal-box__content--MarginRight: var(--pf-c-about-modal-box--PaddingRight);
   --pf-c-about-modal-box__content--MarginBottom: var(--pf-c-about-modal-box--PaddingBottom);
   --pf-c-about-modal-box__content--MarginLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__content--PaddingBottom: var(--pf-c-about-modal-box--PaddingBottom);
+  --pf-c-about-modal-box__content--PaddingTop: var(--pf-c-about-modal-box--PaddingTop);
```

Comment:
You can delete both of these and in the line that references them, use `--pf-c-about-modal-box__content--MarginTop` and `--pf-c-about-modal-box__content--MarginBottom`